include/zephyr/dt-bindings/acpi/acpi.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_ACPI_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_ACPI_H_
+
+#define ACPI_IRQ_DETECT 0xFFFFFFFU
+#define ACPI_IRQ_FLAG_DETECT 0xFFFFFFFU
```

Comment:
I don't see you use these anywhere. I'd suggest to introduce them (and therefore also this file) in a PR where you introduce the first users of them.